html/manifest/protocol_handlers.json

```diff
@@ -12,7 +12,9 @@
             "chrome": {
               "version_added": "96"
             },
```

Comment:
This API was available as a Chrome Origin Trial from versions 92 to 95. Is this something that should be reflected in this file?

Reply:
No, we would only record this in BCD, if the feature could be toggled via a flag/preference.